test/spec/vanilla.spec.js

```
@@ -7,21 +7,17 @@ describe('backend with vanillajs', function() {
     backend.clear();
   });
 
-  it('should attempt to hit end point when mock does not exist', function() {
+  it('should throw an error when no mock has been defined for a request', function() {
     var xhr = new XMLHttpRequest();
-    var response;
-
-    xhr.onreadystatechange = function() {
-      response = xhr.response;
-    };
 
-    xhr.open('GET', 'fixtures/data.json', false);
-    xhr.send();
+    xhr.open('POST', 'api/login', false);
 
-    response.should.be.a('string');
-    response = JSON.parse(response);
-    response.should.be.a('object');
-    response.test.should.equal('hi');
+    try {
+      xhr.send({ username: 'bob', password: 'open-seasame' });
```

I think we should throw an error after this just to avoid false positives